test/Clownface/term.js

```diff
@@ -32,7 +22,7 @@ describe('.term', () => {
     const termA = rdf.literal('1')
     const termB = rdf.namedNode('http://example.org/')
 
-    const cf = clownface(graph, [termA, termB])
+    const cf = clownface({ dataset: rdf.dataset(), terms: [termA, termB] })
```

Comment:
`terms` is intended here and not `term`?

It does not exist at all on the constructor... 🤔

Reply:
Right, fixed it.